.github/workflows/test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Run linters
         uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
         with:
-          version: latest
+          version: v1.64.5
```

Comment:
Locking in specific version of this tool should mitigate breaking changes in newer versions of the lint tool.